clippy.toml

```diff
@@ -0,0 +1,2 @@
+# Require `bevy_ecs::children!` to use `[]` braces, instead of `()` or `{}`.
+standard-macro-braces = [{ name = "children", brace = "[" }]
```

Comment:
I don't think this is a reoccurring problem we have in this codebase, enough to justify adding this when we didn't have it before. If you feel differently, I'm open to hearing if this is really needed.